tests/install/test_install.py

```diff
@@ -3,4 +3,4 @@
 
 def test_install() -> None:
     with pytest.raises(ImportError):
-        import httpx  # type: ignore # noqa
+        import aiohttp  # type: ignore # noqa
```

Comment:
Not sure why we even had it like this, I'll probably rework this in a follow-up.